parser-utils/src/lib.rs

```diff
@@ -8,6 +8,14 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+extern crate combine;
+
+use self::combine::ParseResult;
+use self::combine::combinator::{Expected, FnParser};
```

Comment:
Force of habit! This code came from a module. This — that is, a `lib.rs` file — is the *only* place in Rust you can get away with not using `self::` to qualify an import to the crate you just imported.

`use` expressions are all absolute paths, so if you're in module `foo`, `use thing::bar` means `use ::thing::bar`, not `use foo::thing::bar`.

I'll clean this up.